src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -61,6 +61,7 @@
   private static final String KEY_ERROR_TEXT = "the \"%s\" in the holds is not recognized";
   private static final String INTERVAL_ERROR_TEXT = "the interval \"%s\" in \"%s\" is not recognized";
   private static final String DURATION_ERROR_TEXT = "the duration \"%s\" in \"%s\" is invalid";
+  private static final int DEFAULT_ITEM_LIMIT_VALUE = 0;
```

Comment:
Does this mean a patron cannot borrow any item when limit isn't set?
I think should be discussed with PO because potentially empty value can also mean no limit

Reply:
I agree with @kostyantyn-kh that it should be discussed what no limit means and model that explicitly.

Reply:
updated